### PR TITLE
Bug1588344 5.6

### PR DESCRIPTION
**Pull request by laurynas-biveinis**
http://jenkins.percona.com/job/percona-server-5.6-param/1161/

### CODE REVIEW

File: mysys_ssl/my_default.cc [hunk 1]
```diff
@@ -668,6 +668,7 @@ int my_load_defaults(const char *conf_file, const char **groups,
                                      handle_default_option, (void *) &ctx,
                                      dirs)))
   {
+    delete_dynamic(&args);
```
Comment (percona-ysorokin) [inline mysys_ssl/my_default.cc:672]: Add another "delete_dynamic(&args);" at the code block above (lines 656 - 662)

>    if ((error= my_search_option_files(conf_file, argc, argv,
>                                       &args_used, handle_default_option,
>                                       (void *) &ctx, dirs)))
>    {
>      free_root(&alloc,MYF(0));
>      DBUG_RETURN(error);
>    }

Comment (laurynas-biveinis) [inline mysys_ssl/my_default.cc:672]: Fixing

**Comment by laurynas-biveinis** [comment]
Fixed, http://jenkins.percona.com/job/percona-server-5.6-param/1167/

**Comment by percona-ysorokin** [comment]
LGTM

**Pull request merged**